Livecheckables/akka.rb

```diff
@@ -0,0 +1,6 @@
+class Akka
+  livecheck :url   => "https://github.com/akka/akka.git",
+            # single digit for major version since
+            # v2.6.4 is newer than v10.1.0
+            :regex => /^v?((\d){1}(?:\.\d+)+)$/
```

Comment:
```suggestion
            :regex => /^v?(\d(?:\.\d+)+)$/
```

There are a few things to mention here. If we needed to use a group for `(\d)`, we would want to make it non-capturing like `(?:\d)` (i.e., I like to only have the group for the full version as a capturing group). However, it's not necessary to use a group here and you could simply do `\d{1}`. That said, if you only want to match one digit, you don't need to use `\d{1}` and can simply use `\d`. So, the only change from the standard regex is dropping the `+`, so we're just matching one digit.

Reply:
Sounds good!